test/TestIntegration.java

```diff
@@ -351,6 +353,57 @@ public void appendTwiceBatchedReturnRead() throws Exception {
     assertEquals(write_time, kvts, 5000.0);  // Within five seconds.
   }
 
+  @Test
+  public void basicScanMultipleRegions() throws Exception {
+    splitTable(table, "s10");
+    splitTable(table, "s20");
+    splitTable(table, "s30");
+    List<Deferred<Object>> list = new ArrayList<>();
```

Comment:
The ``<>`` isn't allowed in Java 6. We're still bw compatible for existing users.